plugins/base/src/test/kotlin/translators/DefaultPsiToDocumentableTranslatorTest.kt

```diff
@@ -258,4 +260,56 @@ class DefaultPsiToDocumentableTranslatorTest : BaseAbstractTest() {
             }
         }
     }
+
+    @Disabled // The compiler throws away annotations on unresolved types upstream
+    @Test
+    fun `Can annotate UnresolvedBound`() {
```

Comment:
`UnresolvedBound` are created only in `DefaultPsiToDocumentableTranslator`. 

Here it can be replaced with `unresolved type`.